compiler/include/stmt.h

```diff
@@ -63,13 +63,21 @@ class VisibilityStmt: public Stmt {
 
   virtual ~VisibilityStmt();
 
+  virtual bool isARename() const;
+  virtual const char* getRename() const;
+
+  virtual Symbol* checkIfModuleNameMatches(const char* name);
```

Comment:
I don't think these should be `virtual`. I think they should just be methods on VisibilityStmt and not on UseStmt or ImportStmt.